tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/matter/matters.controller.js

```diff
@@ -254,6 +254,12 @@
                 if (!vm.responseNull) {
                     vm.lazyloader = false;
                     vm.pagenumber = vm.pagenumber + 1;
+                    var finalSearchText = '';
+                    if (vm.selected != undefined) {
+                        finalSearchText = '(' + vm.configSearchContent.ManagedPropertyMatterName + ':"' + vm.selected.trim() + '*" OR ' + vm.configSearchContent.ManagedPropertyMatterId + ':"' + vm.selected.trim() + '*" OR ' + vm.configSearchContent.ManagedPropertyClientName + ':"' + vm.selected.trim() + '*")';

```

Comment:
When doing infinite scroll, we need to use the search term the user has entered in the global search text box